qiita_db/metadata_template/prep_template.py

```diff
@@ -432,6 +404,16 @@ def create_qiime_mapping_file(self, prep_template_fp):
         ------
         ValueError
             If the prep template is not a subset of the sample template
+        QiitaDBWarning
+            If the QIIME-required columns are not present in the template
+
+        Notes
+        -----
+        We cannot ensure that the QIIME-required columns are present in the
+        metadata map. However, we have to generate a QIIME-compliant mapping
+        file. Since the user may need a QIIME mapping file, but not these
+        QIIME-required columns, we are going to create them and
+        populate them with the value XXQIITAXX.
         """
         rename_cols = {
```

Comment:
Oooops! Good call, we still need it, but not as extensive. Instead of:
```python
rename_cols = {
    'barcode': 'BarcodeSequence',
    'barcodesequence': 'BarcodeSequence',
    'primer': 'LinkerPrimerSequence',
    'linkerprimersequence': 'LinkerPrimerSequence',
    'description': 'Description',
}
```
We need to use:
```python
rename_cols = {
    'barcode': 'BarcodeSequence',
    'primer': 'LinkerPrimerSequence',
    'description': 'Description',
}
```
The reason is that we are using different names on the database than on QIIME. Does this seams reasonable? The other solution is to require the user to introduce those values directly. This will require changing the restrictions. What do you think?